Minutes from 2024-01-18 (#1483)
HSF/hsf.github.io: organization/_posts/2024/2024-01-18-coordination.md

---
title: "HSF Coordination Meeting #262, 18 January 2024"
layout: plain_toc
---

Present/Contributing: Graeme Stewart, Pere Mato, Eduardo Rodrigues, Paul Laycock, Benedikt Hegner, Alexander Held, Alexander Moreno, Patrick Gartung, Mark Neubauer, Krzysztof Genser, Michel Jouvin, Nick Smith, Michel Jouvin, Jim Pivarski, Valentin Volkl
  
Apologies/Contributing: Caterina Doglioni

## News, general matters, announcements

### HSF Joint Workshop with WLCG

> The Indico for the WCLG-HSF workshop at DESY (13-17 May) is now available: <https://indico.cern.ch/e/wlcg-hsf-2024>.

Reviewer: ```suggestion
The Indico for the WLCG-HSF workshop at DESY (13-17 May) is now available: <https://indico.cern.ch/e/wlcg-hsf-2024>.
```

Registration should open soon (which will happen via DESY Indico).

At the moment Graeme and Eduardo are "core" organisers (session conveners will be needed). Would anyone else like to join that core team?

- Caterina could help but needs to leave the workshop on May 16th

Ideas for sessions

- We have often had area focused sessions, e.g., simulation, reconstruction, etc.
- Or we go more topical, e.g., accelerators, language evolution
    - For topical sessions, maybe this is a chance to involve the collaboration computing coordinators more in the choice of sessions by asking "what is useful"?
- We could have a session on RSE career panel 
- Should try to link this to EVERSE somehow (kick-off for EVERSE is mid-March)

> At the IRIS-HEP SB (see below), there was support for a session on *Analysis Facilities*.

Reviewer: ```suggestion
At the IRIS-HEP Steering Board meeting (see below), there was support for a session on *Analysis Facilities*.
```

### HSF Organisation and Structure

In the Coordination Team we started to think about how the HSF is organised and how to tackle the challenges for our second decade.

This is an ongoing, and productive, discussion - we should have ideas that we want to discuss more widely with everyone soon.

## Working Group Updates

### Conveners for 2024

We are in the process of inviting new conveners for 2024. We have had quite a few positive responses already:

- Jamie Gooding (Dortmund, LHCb) will join the Data Analysis WG
- Saptaparna Bhattacharya (Wayne State, CMS) will join the Physics Generators WG

> - Nikolai Hartmann (Munchen, ATLAS and Belle II) will join the PyHEP WG

Reviewer: ```suggestion
- Nikolai Hartmann (Munich, ATLAS and Belle II) will join the PyHEP WG
```

### PyHEP

- Organisation of PyHEP.dev 2024 developers workshop due to restart next week.
- That of PyHEP 2024 online user workshop will start concretely once the WG conveners team is set.
- Report on experience gathered over the years at the DESY workshop?

### JuliaHEP

- Graeme gave a talk on Julia at the Grid Deployment Board last week, <https://indico.cern.ch/event/1356130/>.

## Other Interest and Activity Areas

### Google Summer of Code 2024

Registration for organizations opens 22.1. The HSF team will stay the same as last year (Baidyanath, Wouter, Valentin, Benedikt)
Focus this year should be more on AI related topics as asked for by Google. Biggest visible change is the introduction of a small project category. Which means we have small (~90 hours), medium (~175 hrs) and large (~350 hours) projects.

### Analysis Facilities

- The IRIS-HEP steering board organised a [meeting on Analysis Facilities](https://indico.cern.ch/event/1296090/), with presentations from ATLAS, CMS and HSF.
- The meeting was very positive and there was general agreement that we should have a session on this topic at the WLCG-HSF workshop in DESY.

> - In parallel the AFF conveners are making a very concerted effort to finish the [AFF Whitepaper](https://docs.google.com/document/d/1Pn9KWG-tGQ20OaNFUVlXLQddC7vFsQnu2EHR4DBfTjo/edit?usp=sharing), hoping to circulate a second draft in the next few weeks (but if you want to help at this stage, also feel free to take a look).

Reviewer: ```suggestion
- In parallel the Analysis Facilities Forum conveners are making a very concerted effort to finish the [AFF Whitepaper](https://docs.google.com/document/d/1Pn9KWG-tGQ20OaNFUVlXLQddC7vFsQnu2EHR4DBfTjo/edit?usp=sharing), hoping to circulate a second draft in the next few weeks (but if you want to help at this stage, also feel free to take a look).
```
